master/buildbot/db/buildsets.py

```diff
@@ -125,13 +128,13 @@ def thd(conn):
                     mergebrid = artifactbrid = None
 
                 # Add the buildrequest to the database
-                res = conn.execute(ins,
-                                   dict(buildsetid=bsid, buildername=buildername, priority=priority,
-                                        complete=0, results=-1,
-                                        submitted_at=submitted_at, complete_at=None,
-                                        triggeredbybrid=triggeredbybrid, startbrid=startbrid,
-                                        mergebrid=mergebrid, artifactbrid=artifactbrid))
-                brids[buildername] = res.inserted_primary_key[0]
+                with conn_execute(conn, ins,
+                                  dict(buildsetid=bsid, buildername=buildername, priority=priority,
```

Comment:
Could you move this `dict` to a variable?  Don't you think that now the `with` looks scary?

Reply:
Done